server/model/monitor.js

```diff
@@ -475,12 +477,28 @@ class Monitor extends BeanModel {
                 log.debug("monitor", `[${this.name}] sendNotification`);
                 await Monitor.sendNotification(isFirstBeat, this, bean);
 
+                // Set last notified time to now
+                bean.lastNotifiedTime = R.isoDateTime(dayjs.utc());
+
                 // Clear Status Page Cache
                 log.debug("monitor", `[${this.name}] apicache clear`);
                 apicache.clear();
 
             } else {
                 bean.important = false;
+
+                if (bean.status === DOWN && this.resendInterval > 0) {
+                    // divide by 1000 to convert from milliseconds to seconds and divide by 60 to convert from seconds to minutes
+                    let timeSinceLastNotified = (dayjs.utc().valueOf() - (bean.lastNotifiedTime == null ? 0 : dayjs.utc(bean.lastNotifiedTime).valueOf())) / 1000 / 60;
+                    if (timeSinceLastNotified >= this.resendInterval) {
+                        // Send notification again, because we are still DOWN
+                        log.debug("monitor", `[${this.name}] sendNotification again: lastNotifiedTime: ${bean.lastNotifiedTime} | current time: ${R.isoDateTime(dayjs.utc())}`);
+                        await Monitor.sendNotification(isFirstBeat, this, bean);
+
+                        // Set last notified time to now
+                        bean.lastNotifiedTime = R.isoDateTime(dayjs.utc());
```

Comment:
Can you make a variable for this and debug log to have a consistent timestamp?
```suggestion
                        const currentTime = R.isoDateTime(dayjs.utc());
                        log.debug("monitor", `[${this.name}] sendNotification again: lastNotifiedTime: ${bean.lastNotifiedTime} | current time: ${currentTime}`);
                        await Monitor.sendNotification(isFirstBeat, this, bean);

                        // Set last notified time to now
                        bean.lastNotifiedTime = currentTime;
```

Reply:
Thanks that's a good idea 👍